docs/src/app/components/pages/components/Chip/Page.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Title from 'react-title-component';
+
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import chipReadmeText from './README';
+import ChipExampleSimple from './ExampleSimple';
+import chipExampleSimpleCode from '!raw!./ExampleSimple';
+import chipCode from '!raw!material-ui/lib/Chip/Chip';
```

Comment:
Hm, no, you're right. We should probably change that in all the other components